docs/index.html

```diff
@@ -2,7 +2,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
 
 <meta charset="utf-8">
-<meta name="generator" content="quarto-1.0.38">
+<meta name="generator" content="quarto-1.3.433">
```

Comment:
This is current prod Quarto--I just downloaded that version and, at the moment, rendering generates different `site_libs/` output for me. Is this to be expected?

Reply:
I don't think that they would. How are they different?

Reply:
Mostly all `docs/site_libs`.

<img width="797" alt="Screen Shot 2023-07-03 at 3 54 32 PM" src="https://github.com/tidymodels/workshops/assets/35748691/e57d5fa2-5dd5-4950-9aed-7599e207329c">